src/main/java/backend/Season.java

```diff
@@ -0,0 +1,87 @@
+package backend;
+
+import com.google.gson.Gson;
+
```

Comment:
Can you remove the white line? All the imports will be grouped together then. Looks nicer